### PR TITLE
Marketo destination

### PR DESCRIPTION
**Pull request by bleonard**

### CODE REVIEW

File: plugins/@grouparoo/marketo/src/lib/test.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { TestPluginMethod } from "@grouparoo/core";
+import { connect } from "./connect";
+
+export const test: TestPluginMethod = async ({ appOptions }) => {
+  try {
+    const client = await connect(appOptions);
+    const response = await client.lead.describe();
+    const success = response.success ? true : false;
+    return { success };
```
Comment (evantahler) [inline plugins/@grouparoo/marketo/src/lib/test.ts:9]: Is there a message you could return here?  Perhaps something about the account?

**Review by evantahler — APPROVED** [review]
Looks good to me!

**Comment by bleonard** [comment]
Fixes https://github.com/grouparoo/grouparoo/issues/647

File: plugins/@grouparoo/marketo/src/lib/test.ts (hunk 1 above)
Comment (bleonard) [reply plugins/@grouparoo/marketo/src/lib/test.ts:9]: I had trouble finding something useful. I'll make a task.

**Comment by andyjih** [comment]
Closes T-384

**Pull request merged**